src/app/signal/signal-example1/signal-example1.component.html

```diff
@@ -1,6 +1,6 @@
 <h2>Example 1 - Count Increment/Decrement</h2>
 <p>Count is: {{ count() }}</p>
 <p>
-  <button (click)="increaseCount()">Increase</button>
-  <button (click)="decreaseCount()">Decrease</button>
-</p>
\ No newline at end of file
+  <button class="counterButton" (click)="increaseCount()">Increase</button>
```

Comment:
thanks for the PR @Vaibhav-28,
could you please use the [material button](https://material.angular.io/components/button/overview) here